lumicks/pylake/detail/h5_helper.py

```diff
@@ -0,0 +1,39 @@
+import h5py
+
+
+def write_h5(h5_file, output_filename, compression_level=5, omit_data={}):
+    """Write a modified h5 file to disk.
+
+    Parameters
+    ----------
+    h5_file : h5py.File
+        loaded h5 file
+    output_filename : str
+        Output file name.
+    compression_level : int
+        Compression level for gzip compression.
+    omit_data : Set[str]
+        Which data sets to omit. Should be a set of h5 paths.
+    """
+
+    with h5py.File(output_filename, "w") as out_file:
+
+        def traversal_function(name, node):
+            if any([o in name for o in omit_data]):
```

Comment:
I'm afraid that this may be a bit too permissive. That's going to match any substring. That said, I'm not sure I have a better suggestions. Regex is probably overkill, but maybe some simple glob-like matching? I'm sure Python has something builtin for that.

Reply:
Yeah, fair point. Went with `fnmatch` for this (e.g. `Force HF/*`, `*/Force 1y`)

Reply:
Excellent! Please just mention in the parameter docs that it's using `fnmatch` patterns.